contracts/DCAHub/DCAHubPositionHandler.sol

```diff
@@ -324,15 +319,23 @@ abstract contract DCAHubPositionHandler is ReentrancyGuard, DCAHubConfigHandler,
       return _swappedBeforeModified[_positionId];
     }
 
-    uint256 _accumRatio = _userPosition.from < _userPosition.to
-      ? _accumRatio[_userPosition.from][_userPosition.to][_userPosition.swapIntervalMask][_newestSwapToConsider].accumRatioAToB -
-        _accumRatio[_userPosition.from][_userPosition.to][_userPosition.swapIntervalMask][_userPosition.swapWhereLastUpdated].accumRatioAToB
-      : _accumRatio[_userPosition.to][_userPosition.from][_userPosition.swapIntervalMask][_newestSwapToConsider].accumRatioBToA -
-        _accumRatio[_userPosition.to][_userPosition.from][_userPosition.swapIntervalMask][_userPosition.swapWhereLastUpdated].accumRatioBToA;
+    uint256 _positionsAccumRatio;
+    if (_userPosition.from < _userPosition.to) {
+      mapping(uint32 => AccumRatio) storage _accumRatioRef = _accumRatio[_userPosition.from][_userPosition.to][_userPosition.swapIntervalMask];
```

Comment:
fuck this looks awful as a declaration. btw. at the same time, wouldn't you be able to declare `_accumRatioRef` above the `if` statement, avoiding this double declaration?

Reply:
But the thing is that they are used only inside the if, and they are different (one is from-to, and the other is to-from)